lib/generators/plug/install_generator.rb

```diff
@@ -11,5 +11,23 @@ def copy_initializer_file
     def mount_engine
       inject_into_file('config/routes.rb', "\n  mount Plug::Engine => '/plug', as: 'plug'\n\n", before: /^end/)
     end
+
+    def setup_vite_assets
+      copy_file 'plug.vite.js', 'app/frontend/entrypoints/plug.js'
+    end
+
+    def add_plug_package
```

Comment:
This adds a package called "@digitalnz/plug" to the host app during the installation via `rails g plug:install` command.

Reply:
Nice!